app/src/main/java/org/mozilla/focus/settings/PrivacySecuritySettingsFragment.kt

```diff
@@ -0,0 +1,55 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.settings
+
+import android.content.SharedPreferences
+import android.os.Bundle
+import android.preference.Preference
+import android.preference.PreferenceScreen
+
+import org.mozilla.focus.R
+import org.mozilla.focus.telemetry.TelemetryWrapper
+
+class PrivacySecuritySettingsFragment : BaseSettingsFragment(),
+        SharedPreferences.OnSharedPreferenceChangeListener {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        addPreferencesFromResource(R.xml.privacy_security_settings)
+    }
+
+    override fun onResume() {
+        super.onResume()
+
+        preferenceManager.sharedPreferences.registerOnSharedPreferenceChangeListener(this)
+
+        // Update title and icons when returning to fragments.
+        val updater = activity as BaseSettingsFragment.ActionBarUpdater
+        updater.updateTitle(R.string.preference_privacy_and_security_header)
+        updater.updateIcon(R.drawable.ic_back)
+    }
+
+    override fun onPause() {
+        preferenceManager.sharedPreferences.unregisterOnSharedPreferenceChangeListener(this)
+        super.onPause()
+    }
+
+
```

Comment:
\[ktlint\] WARNING: \[no\-consecutive\-blank\-lines\] Needless blank line\(s\) in column 1